packages/design-system/src/components/TextField/style.scss

```diff
@@ -34,6 +34,10 @@ $base-class: 'text-field';
     margin-right: 10px;
   }
 
+  &__label--inline--center {
+    margin-top: 20px;
```

Comment:
please no :) we need to avoid such magic values - it will come back to us sooner or later. I know that the structure of the component makes it hard to handle inline case but margin-top is definitely not the right thing to do here